.gitignore

```diff
@@ -1,7 +1,8 @@
 /node_modules
```

Comment:
This appears here because I added an env.sh file containing the Send Grid key and the Recaptcha key to the gitignore file. However, I'm not sure why it is showing a line that doesn't correspond to the change I made.